src/config/database/postgres/config.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from "@nestjs/common";
```

Comment:
interesting approach to configs but it can be significantly simplified.
here's sample I quickly created as [draft PR](https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-learningSherban/pull/5)

Reply:
it may be not 100% working because I wasn't testing it but it's exact approach we're using on real project